src/audio/multiband_drc/multiband_drc.c

```diff
@@ -182,14 +243,14 @@ static int multiband_drc_init_coef(struct processing_module *mod, int16_t nch, u
 		ret = drc_init_pre_delay_buffers(&state->drc[i], (size_t)sample_bytes, (int)nch);
 		if (ret < 0) {
 			comp_err(dev,
-				 "multiband_drc_init_coef(), could not init pre delay buffers");
+				 "multiband_drc_init_coef(), could not init pre-delay buffers");
 			goto err;
 		}
 
-		ret = drc_set_pre_delay_time(&state->drc[i],
-					     cd->config->drc_coef[i].pre_delay_time, rate);
+		ret = drc_set_pre_delay_time(&state->drc[i], cd->config->drc_coef[i].pre_delay_time,
```

Comment:
this is all formatting and should be done in a different commit or should be dropped